.pre-commit-config.yaml

```diff
@@ -31,19 +33,16 @@ repos:
       - id: mixed-line-ending
         args: ['--fix=lf']
       - id: trailing-whitespace
-  - repo: https://github.com/codespell-project/codespell
```

Comment:
Looks like this completely removed code spell? Was that intentional?

Reply:
Whoops, thanks